src/assets/style/pages/_landing.scss

```diff
@@ -1,18 +1,39 @@
 .landing {
-    .landing__title {
-        font-size: 2rem;
-        font-weight: 700;
-        margin-bottom: 1rem;
-        color: $color-primary;
+    hgroup.landing__title {
+        font-size: 1.5rem;
+        padding: 1rem;
+        filter: drop-shadow(3px 3px 0 rgba(0, 0, 0, .3));
+
+        /* Change the typewriter container layout as needed: */
+        /* Centered: */
+        //text-align: center;
```

Comment:
Remove if not needed

Reply:
It's for future reference thx for your comment tho